.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -122,7 +122,7 @@ Code may not GA until this feature has obtained the `Design Approved` or `No Des
 
 ## **Legal and Translation**
 
-In order to avoid last minute blockers and significant disruptions to the feature, the **legal items need to be done as early in the feature process as possible**, either in design or as early into the development as possible.  Similarly, translation is to be done concurrently with development.  Both **MUST** be completed before Beta or GA is requested.
+In order to avoid last minute blockers and significant disruptions to the feature, the **legal items need to be done as early in the feature process as possible**, either in design or as early into the development as possible.  Similarly, translation is to be done concurrently with development.  All items below **MUST** be completed before Beta or GA is requested.
```

Comment:
The items below are: 

1. Legal (Complete before Feature Complete Date)
2. Innovation (Complete 1 week before Feature Complete Date)
3. Translation (Complete by Feature Complete Date)

Is translation a new requirement for Beta? I thought it was only a requirement for GA?

Reply:
I find the wording confusing if it must be completed by the Feature Complete Date then should we remove the Beta mention in the text above?